app/models/record_mailer.rb

```diff
@@ -7,7 +7,7 @@ def email_record(documents, details, url_gen_params)
             rescue
               I18n.t('blacklight.email.text.default_title')
             end
-    subject = I18n.t('blacklight.email.text.subject', count: documents.length, title: title)
+    subject = I18n.t('blacklight.email.text.subject', count: documents.length, title: title.first)
```

Comment:
Is this guaranteed to be an array?

Reply:
I think yes, unless the user application overrides the title field to be single-valued?

Reply:
Yeah, but they could do that theoretically do that I believe, yeah? (and with something like the canonical title of an item, that might be more common).

Reply:
Yeah, that's true.  I'll take another pass at this.

Reply:
`Array(title).first` should work.